### DISCUSSION TITLE
How can I get get librechat to title a Anthropic conversation?

### DISCUSSION THREAD

**Discussion by EricTheMagician** [danny-avila/LibreChat#1974] [Q&A] [answered]
Hi, I've setup Anthropic and it works well inside of Librechat, but I'm unable to auto-generate the conversation title. It works with the OpenAI conversations.

Does the `TITLE_CONVO` only apply to OpenAI based conversations?

**Comment by dennis8** [comment, marked as answer]
I think this is currently only for OpenAI, but looks like it might be picked up soon:
[Librechat Issue 1422](https://github.com/danny-avila/LibreChat/issues/1422)

**Reply by danny-avila** [reply]
Exactly, this coming week will have it done.

**Reply by EricTheMagician** [reply]
I just updated yesterday and it works now! Thank you!

**Reply by danny-avila** [reply]
Correct, titling is working well! Deeply impressed by Anthropic in this regard (I'm using "function calling" in the manner Claude was trained to use).
